src/test/java/com/zendesk/maxwell/recovery/RecoveryTest.java

```diff
@@ -379,7 +379,8 @@ public void testSchemaIdRestore() throws Exception {
 		LOGGER.info("New pos: " + newPosition);
 		MysqlSavedSchema newSavedSchema = MysqlSavedSchema.restore(context, newPosition);
 		LOGGER.info("New schema id: " + newSavedSchema.getSchemaID());
-		assertEquals(new Long(oldSchemaId + 1), newSavedSchema.getSchemaID());
+		// TODO: this is oldSchemaId +3 insteand of +1 to offset for the maxwell.bootstrap schema migrations and does not seem like a viable long term solution
+		assertEquals(new Long(oldSchemaId + 3), newSavedSchema.getSchemaID());
```

Comment:
ah.  ok, sorry I wasn't quite clear before about the checked in SQL and the schema migrations; please keep *both* the changes to the SQL file and the code that upgrades old schemas.  So new schemas just get the in place schema but old ones are gracefully updated.  then this odd change to this test will go away.